terraform/concourse/outputs.tf

```diff
@@ -64,3 +64,7 @@ output "concourse_db_port" {
   value = aws_db_instance.concourse.port
 }
 
+output "concourse_egress_cidrs" {
+  value = var.concourse_egress_cidrs
```

Comment:
I'm confused about how this is an "output" but we get the value from a "var"

Reply:
Apologies but I'm not sure that I understand the comment. Do you mean that you think that having an output utilise a var is invalid? If so, it is definitely valid terraform and there are many examples pre-existing in this repo e.g.

```
output "bosh_az_label" {
  value = var.zone_labels[var.bosh_az]
}
```
The reason that I added it as an output was that I was having problems removing the concourse egress cidrs in the expunge-concourse job and I wanted to see what the var had been set to in order to debug the issue.